linkchecker

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python -Ru
+#!/usr/bin/python -u
```

Comment:
Better make this
```suggestion
#!/usr/bin/python3 -u
```

Reply:
Thanks, I do forget not everyone has been using Python 3 as python.